src/api/local-api.tsx

```diff
@@ -166,6 +175,26 @@ export class DatacoreLocalApi {
         return this.api.tryEvaluate(expression, variables, sourcePath ?? this.path);
     }
 
+    //////////////
+    // Contexts //
+    //////////////
+
+    // export the necessary contexts to enable rendering
+    // datacore components outside the datacore plugin
+    // itself
+    get SETTINGS_CONTEXT(): typeof SETTINGS_CONTEXT {
```

Comment:
I don't think I'd do it like this - it's better to expose hook functions which load this data instead (like `dc.useSettings()`).